enterprise/api/v1/serializers.py

```diff
@@ -432,46 +433,34 @@ class Meta:
             'enterprise_customer', 'user_email'
         )
 
-    def validate(self, attrs):
-        """
-        Validate if the EnterpriseCustomerUser record already exists.
-        """
-        enterprise_customer = attrs.get('enterprise_customer')
-        user_email = attrs.get('user_email')
-        try:
-            user = User.objects.get(email=user_email)
-            models.EnterpriseCustomerUser.objects.get(
-                user_id=user.pk,
-                enterprise_customer=enterprise_customer
-            )
-        except (User.DoesNotExist, models.EnterpriseCustomerUser.DoesNotExist):
-            pass
-        else:
-            raise serializers.ValidationError('EnterpriseCustomerUser record already exists')
-
-        return attrs
+    def to_representation(self, instance):
```

Comment:
I'm assuming that Django handles this method? I haven't actually used `to_representation` before, although I feel like I could reasonably assume what it does?

Reply:
Yeah, it's a DRF method. It expects an instance of the model to be returned, and creates a JSON serializable representation of it. However, this serializer is very weird and sometimes creates one model, sometimes another, and sometimes nothing at all. And we want to know if something was created or not. So I'm having `create` return both the instance *and* whether or not a user was created, and then telling `to_representation` how to deal with that.

Reply:
Because I have it open, here's what it does. https://github.com/encode/django-rest-framework/blob/master/rest_framework/serializers.py#L493